packages/openmrs/src/Adaptor.js

```diff
@@ -101,24 +61,12 @@ export function getPatient(uuid, callback = false) {
  * @returns {Operation}
  */
 export function createEncounter(data, callback = false) {
-  return state => {
-    const { instanceUrl } = state.configuration;
-    const body = expandReferences(data)(state);
-    const url = `${instanceUrl}/ws/rest/v1/encounter`;
-
-    Log.info(`Creating an encounter.`);
```

Comment:
Same as above... let's keep this logging but use `console.log`, rather than that custom Logger.